packages/node_modules/@ciscospark/test-helper-mocha/src/index.js

```diff
@@ -51,21 +51,23 @@ function noop() {
 
 module.exports = {
   /**
-   * Wrap the desired mochaMethod with `flaky` to indicate it's a flaky test and
-   * exclude it from the cloudapps pipeline
+   * Wrap the desired mochaMethod with `flaky` to indicate it's a flaky test
+   * and exclude it from the test suite.
    * example:
    * `flaky(it)('sometimes passes and sometimes failes')`
    * @param {Function} mochaMethod `it` or `describe`
+   * @param {string} envVar process.env.SKIP_FLAKY_TESTS
    * @returns {Function} mochaMethod or mochaMethod.skip
    */
-  flaky: function flaky(mochaMethod) {
-    // If mochaMethod doesn't have a skip method, assume that mochaMethod is
-    // already either a .skip or a .only
+  flaky: function flaky(mochaMethod, envVar) {
+    // If mochaMethod doesn't have a skip method,
+    // assume the mochaMethod is either a .skip or .only
     if (!mochaMethod.skip) {
       return mochaMethod;
     }
-    // FIXME: These env variables are not available within the browsers.
```

Comment:
![img](https://media0.giphy.com/media/vmtxnxveVUodG/giphy.gif?cid=e1bb72ff5c376cdb416c454a7733e1cc)

Reply:
@adamweeks why is it preferable to have the locks in their own PR?

Reply:
IMO, it helps to prevent merge conflicts between multiple PRs.

Nothing changed in the package.json file, yet the lockfile got updated. It can just cause a lot of merge conflicts if everyone is committing lockfiles with no reason to change it.